samza-rest/src/main/java/org/apache/samza/monitor/LocalStoreMonitorConfig.java

```diff
@@ -46,7 +46,7 @@
    * Defines the comma separated list of job status servers of the form
    * "Host1:Port1,Host2:Port2".
    */
-  private static final String CONFIG_JOB_STATUS_SERVERS = "job.status.servers";
+  private static final String CONFIG_JOB_STATUS_SERVERS = "jobs.status.servers";
```

Comment:
Yes, that wouldn't make less sense to the user (when using both the resources together). 

Is it reasonable to change the prefix of all the config keys in `LocalStoreMonitorConfig` from 'jobs' to 'job' (we'll have consistency across all the config keys in `samza-rest`)